### PR TITLE
Feature/share button

### PR DESCRIPTION
**Pull request by 0x-shanks**
# Overview
* Implementation share button

# Details
* Add Share button
* Add clipboard copy package
* Replace CVoxelDetailBox one

### CODE REVIEW

File: src/components/common/button/shareButton/ShareButton.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      setURL(makeURL());
+    }
+  }, [makeURL, router.isReady, url]);
+
+  const makeTwitterURL = (): string => {
+    const u = new URL("https://twitter.com/intent/tweet");
+    u.searchParams.set("url", url);
+
+    // NOTE: When using additional texts
+    // u.searchParams.set("text", "");
```
Comment (0x-shanks) [inline src/components/common/button/shareButton/ShareButton.tsx]: I was going to put it on, but there was a part I wanted to check, so I left it on.

Is it correct to say "Make Vess"?
Also, if I were to add a hashtag, would it be #Vess?

Comment (kitakaze-kan) [reply src/components/common/button/shareButton/ShareButton.tsx]: Thanks! #Vess is good :) also, how about adding #WorkCredential ? 
For the text part, can you put the below one?
"I made a new Vess on @C_Voxel"

**Review by kitakaze-kan — APPROVED** [review]
LGFM other than the share text part!

File: src/components/common/button/shareButton/ShareButton.tsx (hunk 1 above)
Comment (0x-shanks) [reply src/components/common/button/shareButton/ShareButton.tsx]: fixed: b7606cd

スクリーンショット 2022-06-21 8 19 31

**Pull request merged**